great_expectations/execution_engine/pandas_execution_engine.py

```diff
@@ -318,6 +323,7 @@ def get_batch_data_and_markers(  # noqa: C901 - 22
             buf.seek(0)
             df = reader_fn(buf, **reader_options)
 
+        # ZEP code path
         elif isinstance(batch_spec, PathBatchSpec):
             reader_method = batch_spec.reader_method
             reader_options = batch_spec.reader_options
```

Comment:
Lines `331` and `332` below are where I inject/monkeypatch my Spy for capturing the given `**reader_options`